### DISCUSSION TITLE
Flash.nvim: unknown key <vscode>

### DISCUSSION THREAD

**Discussion by jrock2004** [LazyVim/LazyVim#1891] [Q&A]
So I decided to start over with my lazyconfig since there has been lots of changes to lazy. So I removed all old config. Cleared out cache and plugin dir. Cloned new lazy config from site. Started it up and it installed all the plugins just fine. When I go to check health, I see a warning. Not sure sure if I can ignore and or fix.

```
WARNING {flash.nvim}: unknown key <vscode>
```

**Comment by folke** [comment]
you can ignore that. all good

**Reply by Moanrisy** [reply]
Is there a way to make it OK instead of warning?
or just disable it entirely?

**Reply by Moanrisy** [reply]
It's so weird, this always happening to me :')

Figuring it out for 30m then giving up to ask question
1min later I found the solution

commenting vscode on this work
https://www.lazyvim.org/plugins/editor#flashnvim
` -- vscode = true,`

**Reply by sheeeng** [reply]
> commenting vscode on this work https://www.lazyvim.org/plugins/editor#flashnvim ` -- vscode = true,`

Thank you for the tip, @Moanrisy! 

Modify `${HOME}.local/share/nvim/lazy/LazyVim/lua/lazyvim/plugins/editor.lua` file with the following changes.

```diff
--- a/editor.lua
+++ b/editor.lua
@@ -250,7 +250,8 @@ return {
   {
     "folke/flash.nvim",
     event = "VeryLazy",
-    vscode = true,
+    -- https://github.com/LazyVim/LazyVim/discussions/1891#discussioncomment-7830033
+    -- vscode = true,
     ---@type Flash.Config
     opts = {},
     -- stylua: ignore
```

**Reply by BigJayToDaIzo** [reply]
I gave this workaround a try and the next time I went into LazyVim to update, it gave me sass about the file having manual changes to it and refused to update until I went back and changed the code back.  Frustrating because some of us want to keep that privacy invading junk off our boxes and seeing warnings about it brings about mild discomfort.
